cmd/svcat/broker/register_cmd.go

```diff
@@ -48,27 +68,73 @@ func NewRegisterCmd(cxt *command.Context) *cobra.Command {
 	cmd.Flags().StringVar(&registerCmd.URL, "url", "",
 		"The broker URL (Required)")
 	cmd.MarkFlagRequired("url")
+	cmd.Flags().StringVar(&registerCmd.BasicSecret, "basic-secret", "",
+		"A secret containing basic auth (username/password) information to connect to the broker")
+	cmd.Flags().StringVar(&registerCmd.BearerSecret, "bearer-secret", "",
```

Comment:
Should `basic-secret` and `bearer-secret` not be basic-secret-name and bearer-secret-name?